xclim/indices/helpers.py

```diff
@@ -7,6 +7,8 @@
 """
 from __future__ import annotations
 
+from inspect import stack
+
```

Comment:
```suggestion
import cf_xarray  # noqa
```
To be sure it is imported before accessing  `.cf`.